tests/dummy/app/demo/date-pickers/pickadate/route.js

```diff
@@ -0,0 +1,63 @@
+import Route from '@ember/routing/route';
+import moment from 'moment';
+import { map } from '@ember/object/computed';
+import { computed } from '@ember/object';
+import EmberObject from '@ember/object';
+
+export default Route.extend({
+  model() {
+    const dates = [];
+
+    for (let i = 0; i < Math.round(Math.random() * 305) + 15; i = i + 1) {
```

Comment:
can we explanations for the numbers, 305, 15?

Reply:
pure hack, this library is extremely hard to test, never managed to do better, to be honest..

Reply:
but yeah I can at least try to explain why, lemme add it

Reply:
wait it's the dummy app, so those numbers could be anything I believe? I wouldn't know how to comment that, I guess this choice has been made to make things look nice?

Reply:
added comment